ejb3/src/main/java/org/jboss/as/ejb3/remote/EJBRemoteTransactionPropagatingInterceptor.java

```diff
@@ -114,7 +114,7 @@ private void createOrResumeXidTransaction(final XidTransactionID xidTransactionI
         } else {
             // begin a new tx and add it to the tx repository
             // TODO: Fix the tx timeout to accept a value from the client (WFLY-2789)
-            final Transaction newSubOrdinateTx = this.ejbRemoteTransactionsRepository.importTransaction(xidTransactionID, Integer.MAX_VALUE);
+            final Transaction newSubOrdinateTx = this.ejbRemoteTransactionsRepository.importTransaction(xidTransactionID, Integer.parseInt(System.getProperty("remote-tx-timeout", "31536000")));
```

Comment:
I think having this workardound for EAP 6 CPs is fine. Looking at this from point of view of customer migrating from EAP 6.x CPs to EAP 7, there should be some kind of compatibility to make sure this attribute will be translated to  proper management attribute and the apps using the system property will continue working.